deno.jsonc

```diff
@@ -0,0 +1,17 @@
+{
+  "fmt": {
+    "files": {
+      "exclude": ["./out/"]
+    }
+  },
+  "lint": {
+    "files": {
+      "exclude": ["./out/"]
+    }
+  },
+  "tasks": {
+    "example": "cd examples && deno run --allow-net --allow-read --no-clear-screen --watch deno.ts",
+    "test": "deno test --allow-read --allow-write --ignore=out",
+    "dnt": "deno run --allow-env --allow-net --allow-read --allow-run --allow-write scripts/dnt.ts"
+  }
```

Comment:
```suggestion
  }
  "test": {
    "files": {
      "exclude": ["./out/"]
      }
    }
```

Reply:
Roj: wants to exclude output files during test
Also Roj: fixes Deno runtime to add test config
Also Roj: suggests to use his own config

Reply:
I actually make the contribution because of this project hehe. `deno test` fails when there is dnt's tests in the out folder.